scripts/process_data.py

```
@@ -59,7 +59,7 @@ def save_stats(data_dir: Path, stats_data: dict[str, Any]) -> None:
         logger.warning("処理統計の保存に失敗しましたが、データ処理自体は完了しています")
 
 
-def main() -> None:  # noqa: PLR0912, PLR0915
+def main() -> None:  # noqa: PLR0915
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLR0915`)

Remove unused `noqa` directive

(RUF100)